extensions/amp-story-audio-sticker/amp-story-audio-sticker.md

```diff
@@ -4,8 +4,107 @@ formats:
   - stories
 teaser:
   text: A sticker for users to click and unmute the story. 
+tags:
+  - audio-sticker
+author: ychsieh
+$title: amp-story-audio-sticker
+version: '0.1'
+versions:
+  - '0.1'
+latest_version: '0.1'
+$path: /documentation/components/amp-story-audio-sticker.html
+$scripts:
+  - >-
+    <script async custom-element="amp-story" 
+    src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async custom-element="amp-story-audio-sticker" src="https://cdn.ampproject.org/v0/amp-story-audio-sticker-0.1.js"></script>
 ---
 
+# amp-story-audio-sticker
+
+<amp-video autoplay loop
+            width="400"
+            height="750"
+            layout="fill">
+
+<source src="https://github-production-user-asset-6210df.s3.amazonaws.com/1697814/245924556-e953c54a-0a09-4f15-8134-be6a0ac758ab.mp4" type="video/mp4">
+</amp-video>
+
 # Summary
 
 An audio sticker component for publishers to put anywhere on their story for users to click to unmute the story.
+
+Developers can choose to either choose one of the 4 default stickers or provide custom stickers. The sticker should be put inside an `<amp-story-grid-layer>`.
+
+# Attributes
+
+### `size` {string} optional
+
+Size of the sticker. Accepted values:
+
+-   “large”: 180 x 180 px
+-   “small”: 120 x 120 px
+
+Default value: “small”, which would be used if the input value is invalid or is not provided.
+
+### `sticker` {string} optional
+
+Which of the 4 premade stickers to use. Accepted values:
+
+-   “cat-sticker”
+-   “tape-player”
+-   “loud-speaker”
+-   “audio-cloud”
+
+Default value: “cat-sticker”, which would be used if the input value is invalid or is not provided.
+
+### `sticker-style` {string} optional
+
+Extra premade style of the sticker. Accepted values:
+
+-   “outline”
+-   “dropshadow”
```

Comment:
Explain more. Eg: `outline` draws the outside border. `drop-shadow` surrounds the opaque parts of the sticker.

Also, show an example.

Reply:
Done.